qaframework-bdd-tests/src/test/resources/features/refapp-2.x/10-allergy/allergies.feature

```diff
@@ -1,17 +1,31 @@
 Feature: Allergies Management
 
   Background:
-    Given a user clicks on Allergies link from Patient dashboard
-    Then the system loads Allergies page
+    Given a user clicks on Allergies link from the Patient dashboard
+    Then the system loads the Allergies board page
 
   @selenium
-  @dashboard
-  Scenario: Add No known allergies
-    And a user clicks No Known Allergy button
-    Then the system add no known allergies into the allergies table
-
-  @selenium
-  @dashboard
-  Scenario: Remove No known allergies
-    And a user clicks Remove No Known Allergy icon
+  @allergies
+  Scenario: Allergies management
```

Comment:
what scenario(s) is/are being executed on the allergy feature?

Reply:
@Irenyak1 have you had look at the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/298#discussion_r785394985)?

Reply:
@kdaud the whole test is an allergies management workflow. So the test is doing the full CRUD operations for the both known allergies and unknown allergies.

Reply:
> the whole test is an allergies management workflow

this is captured in the feature title

```js
Feature: Allergies Management
```
do you mind taking a look at the existing Scenario titles within the existing feature files?

Reply:
> > the whole test is an allergies management workflow
> 
> this is captured in the Feature file Title
> 
> ```js
> Feature: Allergies Management
> ```
> 
> do you mind taking a look at the existing Scenario titles within the existing feature files?

@kdaud  I checked out the different feature files and some of them don't necessary align with this one in the way they flow. however one that aligns is the Clinical Visit  management work flow and in it the title is `Feature: Clinical Visit Management` and the scenario is `Clinical Visit ` . So if I compare the two the reason I called the scenario `Allergies Management` it was for lack of a better description of the allergies entire tests. I could not just name it as allergies, may be you could advise me on better words to use cc @sherrif10

Reply:
@Irenyak1 the concern is on the Scenario title that need to renamed so that it tells clearly what the user can do with the allergy feature.

Reply:
@kdaud  ok. However, given that the scenario is handling a bunch of test runs that are separated by user stories, does it mean that I have to make the scenario tile reference all the user stories with in it? For example `Scenario: Add No known allergies, Remove No known allergies, Add known allergy, Edit known allergy and Delete known allergy`.  To me that does not look smart.

Reply:
@Irenyak1 have you taken consideration of

```js 

Feature: Allergies Management

Scenario: Allergies Management
?

have you taken a look at? https://automationpanda.com/2017/01/30/bdd-101-writing-good-gherkin/

Reply:
> @Irenyak1 have you taken consideration of
> 
> ```js
> Feature: Allergies Management
> 
> Scenario: Allergies Management
> ?
> ```
> 
> have you taken a look at? https://automationpanda.com/2017/01/30/bdd-101-writing-good-gherkin/

@kdaud  sometimes it is very good and time saving to be direct with what you are meaning. Some reviews are full of questions that leave the developer wondering what you are intending to mean. For this case if you had initially suggested this change, I would straight away know that you are implying that I did not use caps for m in `management` , But also it is not necessarily that `Scenario: Allergies Management` must have a capital M.

I have just picked one acceptable example from the document you shared. 

`Feature: Google Searching

  Scenario: Search from the search bar
    Given a web browser is at the Google home page
    When the user enters "panda" into the search bar
    Then links related to "panda" are shown on the results page

  Scenario: Image search
    Given Google search results for "panda" are shown
    When the user clicks on the "Images" link at the top of the results page
    Then images related to "panda" are shown on the results page ` 

Unless there's a change I have not captured. I hope I have understood you properly. If not please clarify to me.

Reply:
Scenario title highlights what is being performed on the feature so would be a good idea revisit the title.

Reply:
@kdaud  I have pushed a commit with the scenario titles renamed. Does it look better?